copyspecial.py

```diff
@@ -6,36 +6,108 @@
 # Google's Python Class
 # http://code.google.com/edu/languages/google-python-class/
 
-import sys
 import re
 import os
 import shutil
 import commands
 import argparse
+from zipfile import ZipFile
+"""Copy Special exercise"""
 
-"""Copy Special exercise
-"""
+# change this for a
+SPECIAL_RE = re.compile(r'.+__\w+__.+')
+
+
+def get_special(dirname):
+    #  returns a list of the absolute paths of
+    #  the special files in the given directory
+    global SPECIAL_RE
+    sp_list = filter(lambda f: bool(SPECIAL_RE.match(f)), os.listdir(dirname))
+    return map(lambda f: os.path.join(os.getcwd(), f), sp_list)
+
+
+def copy_to(files, dest):
+    # given a list of paths,
+    # copies those files into the given directory
+    for file in files:
+        shutil.copyfile(file, os.path.join(dest, file.split('/')[-1]))
+    return
```

Comment:
You don't need to use return if nothing is being returned. Functions automatically return None if return isn't explicitly called.

Reply:
Thanks for the note!